airflow/providers/google/cloud/transfers/bigquery_to_gcs.py

```diff
@@ -139,14 +139,34 @@ def execute(self, context):
             location=self.location,
             impersonation_chain=self.impersonation_chain,
         )
-        conn = hook.get_conn()
-        cursor = conn.cursor()
-        cursor.run_extract(
-            source_project_dataset_table=self.source_project_dataset_table,
-            destination_cloud_storage_uris=self.destination_cloud_storage_uris,
-            compression=self.compression,
-            export_format=self.export_format,
-            field_delimiter=self.field_delimiter,
-            print_header=self.print_header,
-            labels=self.labels,
+
+        source_project, source_dataset, source_table = split_tablename(
```

Comment:
Let us use `TableReference.from_string` instead of custom method
```
>>> from google.cloud.bigquery.table import TableReference
>>> TableReference.from_string('my-project.mydataset.mytable')
TableRef...(DatasetRef...('my-project', 'mydataset'), 'mytable')
```